client/landing/stepper/declarative-flow/internals/global.scss

```
@@ -287,3 +287,12 @@ button {
 		}
 	}
 }
+
+.ecommerce {
```

I think this should be `.ecommerce.processing` to match the selectivity of the original rule.
```suggestion
.ecommerce.processing {
```